### PR TITLE
Addresss app-wiki issue #11 - fix linter settings.

### PR DESCRIPTION
**Pull request by Eckhardt-D**
Fix invalid value for property - `'no-use-before-define'` from 'on' to '1' which is a warning level setting.

### CODE REVIEW

**Review by Eckhardt-D — COMMENTED** [review]
Is it necessary to commit the .expo file? should we not add it to .gitignore ?

**Comment by Eckhardt-D** [comment]
[Issue #11 Invalid eslintrc property value.](https://trello.com/c/xiunmy6Q/45-issue-11-invalid-eslintrc-property-value)

File: .eslintrc.js [hunk 1]
```diff
@@ -8,7 +8,7 @@ module.exports = {
     'jest': true,
   },
   'rules': {
-    'no-use-before-define': 'on',
+    'no-use-before-define': 1,
```
Comment (snordmann) [inline .eslintrc.js]: please set this to `"error"`. Currently it is set to `1` which means only warning. Setting it to error will ensure a better code quality and will be more semantic than writing `2`.

**Review by snordmann — CHANGES_REQUESTED** [review]

**Review by snordmann — APPROVED** [review]
/lgtm

**Pull request merged**